### PR TITLE
[#566] Add meta-schema.json and test using it.

### PR DESCRIPTION
**Pull request by kindly**
related #566

### CODE REVIEW

File: standard/schema/meta-schema.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                    "minItems": 1,
+                    "uniqueItems": true
+                }
+            ]
+        },
+        "allOf": { "$ref": "#/definitions/schemaArray" },
+        "anyOf": { "$ref": "#/definitions/schemaArray" },
+        "oneOf": { "$ref": "#/definitions/schemaArray" },
+        "not": { "$ref": "#" },
+        "$ref": {
```
Comment (jpmckinney) [inline standard/schema/meta-schema.json]: For `$ref`, draft-06 has:

```
        "$ref": {
            "type": "string",
            "format": "uri-reference"
        },
```

though we'd use `"format": "uri"` for draft-04.

File: standard/schema/meta-schema.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        },
+        "versionId": {
+            "type": "boolean",
+            "default": false
+        },
+        "wholeListMerge": {
+            "type": "boolean",
+            "default": false
+        }
+
```
Comment (jpmckinney) [inline standard/schema/meta-schema.json]: Extra line.

**Review by jpmckinney — COMMENTED** [review]
Made a few comments. It might be good for easier change-tracking to first commit the draft-04 schema, and then commit our changes, so that we can easily see the divergence when reviewing history.

File: standard/schema/meta-schema.json (hunk 1 above)
Comment (kindly) [reply standard/schema/meta-schema.json]: I think the issue here is that internal $ref i.e '#/definitions/Awards' would not pass "uri" validataion. Which I think is why they added the new format.

Comment (jpmckinney) [reply standard/schema/meta-schema.json]: Aha, makes sense.

File: standard/schema/tests/test_validate_schema.py [hunk 3]
```diff
@@ -31,14 +31,15 @@ def object_pairs_hook(pairs):
 
 
 DRAFT_SCHEMA_PATH = path(__file__).parent / 'json-schema-draft-4.json'
```
Comment (jpmckinney) [inline standard/schema/tests/test_validate_schema.py]: This constant's value needs to be updated because the file moved in another commit, but if we aren't using the constant anymore, we can just remove it.
